README.md

```diff
@@ -38,10 +43,10 @@ sample_df = train_df.sample(frac=0.01, random_state=0)
 sample_df.sort_values("AvSigVersion", inplace=True)
 
 # define the validation scheme
-cv = KFold(n_splits=4, shuffle=False, random_state=0)
+cv = KFold(n_splits=4, shuffle=True, random_state=None)
```

Comment:
Maybe it changed with a new sklearn version. Anyway, you can set the random_state to None but shuffle should stay as False since it is doing a lazy time split.

Reply:
Yep, it seems like the API has changed.

Reply:
Maybe good to add a comment and make the example a bit more general:

`sample_df.sort_values("AvSigVersion", inplace=True)`
->
`sample_df = sample_df.sample(frac=1)  # Shuffling rows before CV`

and
`cv = KFold(n_splits=4, shuffle=False)  # No shuffling to keep the same folds for each feature`

Reply:
In this example, we dont shuffle because the data is sorted by time. It is a lazy way to do time split validation instead of a proper sliding windows approach.

When seed is given, we have the same folds for all the features. It wasn't the reason for shuffle=False.

Reply:
So best would be to remove:
`sample_df.sort_values("AvSigVersion", inplace=True`
and put only
`cv = KFold(n_splits=4, shuffle=True, random_state=0)  # Fix random_state to keep the same folds for each feature`
right?

Reply:
Good point @stephanecollot 

Alternatively, we can use :
`cv = KFold(4, shuffle=False, random_state=None)`

Problem exists only when we have `random_state` set to `int` and `shuffle` False

Reply:
We want to keep the data sorted by AvSigVersion for this problem and split it by AvSigVersion. AvSigVersion is like a proxy for time. Just setting random_state to None without changing anything else would be the best.

Reply:
The issue @aerdem4 is this data is not available in the repo and that makes a very confusing readme since no body know this column AvSigVersion. And having a time split validation is maybe not the best readme example you want to show.

Reply:
I thought anyone could search for the competition name and get the data but we can also share the link to the competition on readme? We can also comment that AvSigVersion is a proxy for time because the data had no time column.

> And having a time split validation is maybe not the best readme example you want to show. 
Why?

Reply:
@KameniAlexNea can you please add these 2 comments above that @stephanecollot shared since you already update the readme? Then I can merge it.

Reply:
I just make a push now